src/client/java/teammates/client/scripts/PlaceholderUserScript.java

```diff
@@ -0,0 +1,15 @@
+package teammates.client.scripts;
+
+/**
+ * Placeholder Script to configure execScript task when not run by user.
```

Comment:
This class description sounds a bit weird. What does it mean `when not run by user`?

Reply:
When the user runs `./gradlew execScript -PuserScript=MyUserScript` this class is not needed. If a user was to perform another Gradle task such as `./gradlew tasks`, `execScript` is still configured and since the variable `userScript` variable is `null` the task would fail.

Therefore this class is a placeholder to configure Gradle when the user does not run the task `execScript`

Reply:
Could try what samuel suggested below. Seems to be a better way

Reply:
Will be using his suggestion in the latest commit